packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -49,7 +49,7 @@ Requires: %{?scl_prefix}rubygem(runcible) >= 2.13.0
 Requires: %{?scl_prefix}rubygem(runcible) < 3.0.0
 Requires: %{?scl_prefix}rubygem(anemone)
 Requires: %{?scl_prefix}rubygem(pulpcore_client) >= 3.6.0
-Requires: %{?scl_prefix}rubygem(pulpcore_client) < 3.8.0
+Requires: %{?scl_prefix}rubygem(pulpcore_client) < 3.7.7
```

Comment:
This  line was updated to reflect this PR https://github.com/Katello/katello/pull/9493